cognite/client/_api/templates.py

```diff
@@ -26,6 +27,13 @@
     from cognite.client.config import ClientConfig
 
 
+def templates_deprecation_warning() -> None:
+    return warnings.warn(
+        "Templates will be removed in future versions of the SDK. Please migrate to Flexible Data Modeling.",
```

Comment:
This sounds better to me as a norwenglishman 😂  
```suggestion
        "Templates will be removed in a future version of the SDK. Please migrate to Data Modeling. "
        "Read more at: https://docs.cognite.com/cdf/data_modeling/",
```

Reply:
I actually had written that suggestion initially, but I decided to copy what was written in some other deprecation messages elsewhere. Who knows, maybe Templates come back (not likely, but y'know what I mean).